app/src/forms/minesoperatorscreening/constants.js

```diff
@@ -17,5 +17,6 @@ module.exports = Object.freeze({
   SLEEPING_AREA_TYPE_SHARED: SLEEPING_AREA_TYPE_SHARED,
   SLEEPING_AREA_TYPES: SLEEPING_AREA_TYPES,
   UUID_REGEX: '^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}$',
-  CONFIRMATION_ID_REGEX: '^[0-9A-Fa-f]{8}$'
+  CONFIRMATION_ID_REGEX: '^[0-9A-Fa-f]{8}$',
+  MINE_NUMBER_REGEX: '^[0-9]{7}$',
```

Comment:
Will prevent them from entering "12345" if their mine number is "0012345" but I think that's the point so will restrict the same way on the FE.

Reply:
matt said 0 padded...

Reply:
Oh ok, never saw that. Yeah I'll 0 pad it